systest/build.rs

```diff
@@ -34,5 +34,6 @@ fn main() {
         }
     });
     cfg.skip_type(|t| t == "__enum_ty");
+    cfg.skip_field_type(|s, field| s == "git_rebase_operation" && field == "id");
```

Comment:
Could you add a TODO here pointing at the ctest issue?

Reply:
Done.